src/routes/operator.ts

```diff
@@ -302,9 +303,18 @@ class QuoteCache {
           }
           delete this.pending[key];
 
-          if (result.success && result.expires === undefined) {
-            // Default to 60 seconds expiry
-            result.expires = new Date(Date.now() + 60_000);
+          if (result.success) {
```

Comment:
I had to update this logic to also account for bogus dates. For some reason mayan quote api is returning "0" which creates a date object with 1969. As such === undefined check is not sufficient to guard against bad dates.

Reply:
Does Mayan know about this?

Reply:
Looks like this is as intended